### DISCUSSION TITLE
How to disable the popup about file not checked?

### DISCUSSION THREAD

**Discussion by ronakg** [streetsidesoftware/vscode-spell-checker#1428] [Q&A] [answered]
This popup shows up every time I start VSCode. I know that the file is not being checked for spell errors. Is there a way to disable this popup?

![image](https://user-images.githubusercontent.com/1058322/136993314-4fb5b02c-9b24-4045-b2b8-56f75776aa0e.png)

**Comment by Jason3S** [comment, marked as answer]
The pop-up is going to be removed in the next release.
